Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerRawDataManager.swift

```diff
@@ -239,10 +239,10 @@ extension PrimerHeadlessUniversalCheckout {
                             
                         case .continueWithNewClientToken(let newClientToken):
                             firstly {
-                                ClientTokenService.storeClientToken(newClientToken)
+                                ClientTokenService.storeClientToken(newClientToken, isAPIValidationEnabled: true)
                             }
                             .then { () -> Promise<Void> in
-                                let configurationService = PrimerAPIConfigurationService(requestDisplayMetadata: true)
```

Comment:
Noticing it here as well. I'm sure it does then 👍

Reply:
Nope, as above. It does make the request faster.